v21.1/backup.md

```diff
@@ -23,42 +23,52 @@ You can also back up:
 - [An individual database](#backup-a-database), which includes all of its tables and views
 - [An individual table](#backup-a-table-or-view), which includes its indexes and views
 
-Because CockroachDB is designed with high fault tolerance, these backups are designed primarily for disaster recovery (i.e., if your cluster loses a majority of its nodes) through [`RESTORE`](restore.html). Isolated issues (such as small-scale node outages) do not require any intervention.
+    `BACKUP` only offers table-level granularity; it _does not_ support backing up subsets of a table.
 
-{{site.data.alerts.callout_info}}
-`BACKUP` only offers table-level granularity; it _does not_ support backing up subsets of a table.
-{{site.data.alerts.end}}
+Because CockroachDB is designed with high fault tolerance, these backups are designed primarily for disaster recovery (i.e., if your cluster loses a majority of its nodes) through [`RESTORE`](restore.html). Isolated issues (such as small-scale node outages) do not require any intervention.
 
 {{site.data.alerts.callout_success}}
 To view the contents of an enterprise backup created with the `BACKUP` statement, use [`SHOW BACKUP`](show-backup.html).
 {{site.data.alerts.end}}
 
 ## Required privileges
 
-- Only members of the `admin` role can run `BACKUP`. By default, the `root` user belongs to the `admin` role.
+- Full cluster backups can only be run by members of the `admin` role. By default, the `root` user belongs to the `admin` role.
+- For all other backups, the user must have read access (`SELECT` or `USAGE`) on all objects being backed up.
 - `BACKUP` requires full read and write (including delete and overwrite) permissions to its target destination.
 
+### Destination privileges
+
+{% include {{ page.version.version }}/backups/destination-file-privileges.md %}
+
 ## Synopsis
 
 <div>
 {% include {{ page.version.version }}/sql/generated/diagrams/backup.html %}
 </div>
 
+{{site.data.alerts.callout_info}}
```

Comment:
~Is there some doc we can link to describing this "collection" concept?
~Suggest saying explicitly "The BACKUP ... TO syntax is no longer supported", if that is what we are implying here (?).  If you are breaking me I want to know right away up front that I need to do something.  If you are not breaking me, but deprecating the syntax in favor of $NEW_HOTNESS, I *strongly* suggest we use the word "deprecated", which has a specific meaning
~Suggest editing "previous versions" to say "versions of CockroachDB prior to v21.1, ..." since the "new in v21.1" tag will go away at some point, and a specific version is needed to refer to
~It feels like this is a lot of information to get into one paragraph.  Perhaps we are trying too hard to cram it because it's in a note? Also, I feel like this should be up near the top of the page, not down here in the Synopsis (especially if we are actually taking BACKUP ... TO away entirely)
~Strongly recommend not using "old"; suggest saying something like "for documentation on the no-longer-supported BACKUP ... TO syntax, see the docs for v20.2 ..." (if it's no longer supported, of course - see question elsewhere)
~Main feedback: please tell me directly what is happening here.  Is the new thing "in addition to" or "instead of"?  If it's the latter I'm gonna be really frustrated if i have to parse that out from subtle, implied language.  If we are deprecating but don't use the word "deprecating" I'm also gonna be frustrated.
~Postscript: Sorry if the above is a bit aggressive, I'm the type of user who gets *real squirrelly* when software I use changes, because I don't like changing things if they already "work for me (tm)".  I get double squirrelly if I don't understand from the docs what exactly is changing, and exactly when.